data/json/terrain.json

```diff
@@ -3156,9 +3156,29 @@
         "id" : "t_tree_maple",
         "name": "maple tree",
         "symbol": "7",
-        "color": "red",
+        "color": [ "light_green", "green", "red", "brown" ],
```

Comment:
A maple tree is not harvestable. It can be tapped during late winter and early spring. What color should it have?